### ISSUE TITLE
NASNet Model

### ISSUE THREAD

**Issue by ahkarami** [moskomule/senet.pytorch#4]
Dear @moskomule,
Thank your for your nice repository. Recently the Google Brain Team released a fantastic CNN model, [NASNet](https://arxiv.org/abs/1707.07012), in [TF-slim](https://github.com/tensorflow/models/tree/master/research/slim/nets/nasnet), which achieved the state-of-the-art Top-1 Accuracy on ImageNet by **82.7 %**. According to your ability by implementing the SENet Model in PyTorch, would you please kindly implement or port this model into PyTorch?

**Comment by moskomule** [comment]
Hi @ahkarami,
I'm not sure if I can implement it or not but I'll try.

**Comment by ahkarami** [comment]
Thank you very much.
